core/src/main/java/io/questdb/MessageBus.java

```diff
@@ -93,4 +93,10 @@ default void close() {
     RingQueue<VectorAggregateTask> getVectorAggregateQueue();
 
     Sequence getVectorAggregateSubSeq();
+
+    MPSequence getQueryCacheEventPubSeq();
+
+    RingQueue<QueryCacheTask> getQueryCacheEventQueue();
```

Comment:
not used, queue is not needed. The existing functionality is implemented using sequences

Reply:
Good point. Removed the queue in 72725c4594e4ebd2d80f4854d2c1d795cc864687